src/main/java/com/springweb/dv_spring_web_mongo/exception_handling/ProjectIncorrectData.java

```diff
@@ -0,0 +1,13 @@
+package com.springweb.dv_spring_web_mongo.exception_handling;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
you can remove `access = AccessLevel.PUBLIC` besause the value is PUBLIC by default